src/applications/disability-benefits/disability-rating-calculator/components/DisabilityRatingCalculator.jsx

```diff
@@ -44,21 +42,40 @@ export default class DisabilityRatingCalculator extends React.Component {
   focusLastRatingInput = () =>
     this.ratingInputRefs[this.ratingInputRefs.length - 1].focus();
 
+  focusFirstInvalidInput = () => {
+    const firstInvalidDisability = this.state.disabilities.findIndex(
+      d => d.hasError,
+    );
+    this.ratingInputRefs[firstInvalidDisability].focus();
+  };
+
   handleDisabilityChange = (index, updatedRow) => {
     const disabilities = this.state.disabilities;
     disabilities[index] = updatedRow;
     this.setState({ disabilities });
   };
 
-  handleSubmit = event => {
-    event.preventDefault();
-
+  handleSubmit = () => {
     const ratings = getRatings(this.state.disabilities);
-    const calcRating = calculateCombinedRating(ratings);
+    const disabilitiesValidated = this.state.disabilities.map(disability => ({
+      ...disability,
+      hasError: !isRatingValid(disability.rating),
+    }));
```

Comment:
We calculate input errors from this parent component (instead of in each `RatingRow`) so that we can easily stop the form submission by just checking if any element in `this.state.disabilities` has `hasError` set to true.